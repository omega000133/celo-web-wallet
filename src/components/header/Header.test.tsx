// import { renderWithProvider } from 'src/test/utils'
// import { Header } from './Header'

describe('Header', () => {
  it('matches snapshot', () => {
    // const { getByText } = renderWithProvider(<Header />)
    // const address = getByText(NULL_ADDRESS)
    // expect(address).toBeTruthy()
  })
})
