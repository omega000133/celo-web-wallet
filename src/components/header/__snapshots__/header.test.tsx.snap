// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1
      class="css-pm4pww-Header"
    >
      Your address is 0x0000000000000000000000000000000000000000
    </h1>
    <button>
      Create New Wallet
    </button>
  </div>
</DocumentFragment>
`;
